years.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+if (grep -r "Copyright \+(c) \+2009-.*rultor.com" \
+  --exclude-dir "target" --exclude-dir ".git" --exclude "years.sh" . \
+  | grep -v 2009-`date +%Y`) then
+    echo "Files above have wrong years in copyrights"
+    exit 1
+fi
+
+if (grep -r -L "Copyright \+(c) \+2009-.*rultor.com" \
+ --include=*.{java,xml,vm,groovy,txt,fml,properties} --exclude-dir "target" . \
+ | grep -v 2009-`date +%Y`) then
```

Comment:
@mkordas 

> @vkuchyn I've removed couple of licenses from Java and XML code and when running ./years.sh nothing is reported to me

It is very strange - I get violations if to remove copyright license line or to change year. I will now make commit to check it on travis. Did you remove whole line or changed years?

> @vkuchyn I still think that this last grep is not needed
 
Also will check on travis. What OS do you have? Cause I've noticed grep works in different way on mac os